src/Network/Request.php

```diff
@@ -1161,11 +1161,15 @@ protected function _parseAcceptWithQualifier($header)
      * Provides a read accessor for `$this->query`. Allows you
      * to use a syntax similar to `CakeSession` for reading URL query data.
      *
-     * @param string $name Query string variable name
+     * @param string|null $name Query string variable name or null to read all.
      * @return mixed The value being read
```

Comment:
shouldn't this be `@return string|null`. Or do we support array values in query strings. I can't remember.